jobfair-frontend/src/components/forms/ResetPasswordForm/ResetPasswordForm.component.jsx

```diff
@@ -25,30 +32,31 @@ const ResetPasswordFormComponent = ({ onFinish, form, email, setOtpCode }) => {
             <Typography variant='h6' gutterBottom component='div'>
               OTP Code:
             </Typography>
-            <ReactCodeInput id='pinCode' type='text' fields={6} onChange={handlePinChange} />
+            <ReactCodeInput id='pinCode' type='number' fields={6} onChange={handlePinChange} autoComplete={'false'} />
```

Comment:
why 'false' not false ?

Reply:
autoComplete props receives type string, so it can be any string.
![image](https://user-images.githubusercontent.com/62231837/172055077-cba05968-a6e5-495c-b8b6-2c9b53e42fea.png)